src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,64 @@
+package duke;
```

Comment:
Overall, I think it is a really good job, and I especially like the separation of commands to make each file much more manageable!